Rakefile

```diff
@@ -10,12 +10,9 @@ task default: [:ci]
 
 desc 'Continuous integration task run on travis'
 task ci: [:environment, :rubocop] do
-  if Rails.env.test?
-    Rake::Task['db:setup'].invoke # not db:migrate, use the dang schema!
-    Rake::Task['spec'].invoke
-  else
-    system 'rake ci RAILS_ENV=test'
-  end
+  raise 'Refusing to run CI with RAILS_ENV=production' if Rails.env.production?
+  Rake::Task['db:setup'].invoke # not db:migrate, use the dang schema!
```

Comment:
Could we explicitly set the `ENV['RAILS_ENV'] = test` here or does it have to be set in the shell and that's why the existing code calls back to `system`?

Reply:
Yeah, it doesn't work like that, because of timing issues.  Since the rails environment is already initialized by this point, changing the ENV does **not** reinitialize rails.